### PR TITLE
copy samples for 14.0.0 release

### PR DESCRIPTION
**Pull request by bbzurek**
there are still some raml spec projects, are they referenced anywhere?

### CODE REVIEW

File: service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>backbase-spec-starter-parent</artifactId>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <version>13.0.0</version>
```
Comment (bb-samp) [inline service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml]: Sorry, just checking, is this meant to be 14.0.0? thanks

File: service-sdk/14.0.0/create-persistence-crud-service-specification/example-service-spec/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>backbase-spec-starter-parent</artifactId>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <version>13.0.0</version>
```
Comment (bb-samp) [inline service-sdk/14.0.0/create-persistence-crud-service-specification/example-service-spec/pom.xml:9]: (As below) sorry, just checking if this is meant to be 14.0.0? thanks

File: service-sdk/14.0.0/create-outbound-integration-service/example-integration-spec/pom.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>backbase-spec-starter-parent</artifactId>
+        <groupId>com.backbase.buildingblocks</groupId>
+        <version>13.0.0</version>
```
Comment (bb-samp) [inline service-sdk/14.0.0/create-outbound-integration-service/example-integration-spec/pom.xml:9]: (As below) sorry, just checking if this is meant to be 14.0.0? thanks
(Apologies for the comment spam)

File: service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml (hunk 1 above)
Comment (bbzurek) [reply service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml]: 14.0.0 doesn't have the plugin referenced in this pom, hence my question in the PR description

Comment (SteveWyatt74) [reply service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml]: RAML specs are no longer referenced in Community.

Comment (SteveWyatt74) [reply service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml]: I'll get a list together of what projects can be kept/ deleted

Comment (SteveWyatt74) [reply service-sdk/14.0.0/migrate-to-open-api/message-service-spec-raml/pom.xml]: migrate-to-open-api directory can be deleted, rest are still used by Community

**Pull request merged**